kahuna/public/js/components/gu-lazy-table-shortcuts/gu-lazy-table-shortcuts.js

```diff
@@ -0,0 +1,62 @@
+import angular from 'angular';
+
+import '../gr-keyboard-shortcut/gr-keyboard-shortcut';
+
+export var lazyTableShortcuts = angular.module('gu.lazyTableShortcuts', [
+    'gr.keyboardShortcut'
+]);
+
+lazyTableShortcuts.directive('guLazyTableShortcuts', ['keyboardShortcut', function(keyboardShortcut) {
+    return {
+        restrict: 'EA',
+        require: '^guLazyTable',
+        link: function (scope, element, attrs, lazyTableCtrl) {
```

Comment:
I quite like this pattern of implementing extra functionality by tapping into public controller methods exposed by a parent directive, which I stole from elsewhere of course.

cc @hmgibson23 would be nice for Scribe + plugins!